addons/test_website/tests/test_reset_views.py

```diff
@@ -17,8 +18,14 @@ def fix_it(self, page, mode='soft'):
         resp = self.url_open(page)
         self.assertEqual(resp.status_code, 500, "Waiting 500")
         self.assertTrue('<button data-mode="soft" class="reset_templates_button' in resp.text)
-        data = {'view_id': self.find_template(resp), 'redirect': page, 'mode': mode}
-        resp = self.url_open('/website/reset_template', data)
```

Comment:
This is wrong, the test is not testing anything anymore, as it's not testing that accessing the page works fine (return 200).
It's now testing that the reset controller is working fine (return 200).
You can see it by checking `resp.url` before and after this commit, before it's the /test_page_view page, after it's the /reset/template controller
(I mean, this line is ok, it just need to now open the page itself and store the response in `resp` for the next check to check the code on the page and not the controller)